akka-docs/rst/java/lambda-persistence.rst

```diff
@@ -138,13 +138,9 @@ the ``persist`` call and the execution(s) of the associated event handler. This
 calls in context of a single command.
 
 If persistence of an event fails, ``onPersistFailure`` will be invoked (logging the error by default)
-and the actor will unconditionally be stopped. The reason that it cannot resume when persist fails
-is that it is unknown if the even was actually persisted or not, and therefore it is in an inconsistent
-state. Restarting on persistent failures will most likely fail anyway, since the journal is probably
-unavailable. It is better to stop the actor and after a back-off timeout start it again. The
-``akka.persistence.BackoffSupervisor`` actor is provided to support such restarts.
-
-.. includecode:: ../../../akka-samples/akka-sample-persistence-java-lambda/src/main/java/doc/LambdaPersistenceDocTest.java#backoff
+and the actor will unconditionally be stopped. If persistence of an event is rejected before it is
+stored, e.g. due to serialization error, ``onPersistRejected`` will be invoked (logging a warning 
+by default) and the actor continues with next message.
```

Comment:
yes, sounds good - takes care of the "my datastore does the validation" case we had touched upon in one issue.